Interfaces/API/Dirac.py

```diff
@@ -872,8 +872,9 @@ def runLocal(self, job):
           elif os.path.isdir( isFile ):
             self.log.debug("Input Sandbox %s is a directory, found in the user working directory, copying it" % isFile)
             shutil.copytree( isFile, os.path.basename( isFile ), symlinks = True )
-          elif os.path.exists( isFile ):
```

Comment:
This check is already done before the else, because it checks if the file is in the current working directory.
Also the following string says something about copying while it should say it is already in the current working directory